README.md

```diff
@@ -2,76 +2,32 @@
-
-then you should contact an admin and ask them to add you to the Google Drive hunt folder. For more details, see the [hunt setup guide](new-hunt-setup.md#giving-a-new-user-access-to-cardboard).
-
-### Adding and modifying puzzles
-
-Any user can add a new puzzle by using the blue "Add New Puzzle" button in the top right.
-
-<img src='https://user-images.githubusercontent.com/544734/71759777-3cdc8d00-2e80-11ea-9d49-48de77370976.png' width='300'>
-
-You must specify a name but the URL is optional. If you know the puzzle is a meta puzzle, you can check the "Meta?" checkbox. In case you made a mistake or realize a puzzle is a meta later on, you can always go back and edit a puzzle by pressing the pencil icon next to the puzzle name.
-
-In case you need to delete a puzzle (perhaps you accidentally created a duplicate), you can do so using the trash icon next to a puzzle name.
-
-When a puzzle is created, a Google Spreadsheet are automatically created for the puzzle. These are linked from the Sheets icon.
-
-### Searching
-
-As a hunt progresses, the number of puzzles can get quite large (especially for a hunt like the MIT Mystery Hunt). To make things more manageable, you can use the "Search" box in the top-right to filter puzzles. The search matches puzzle name, answer, status, or tags. For example, the following shows filtering to just the puzzles that need extraction:
-
-![extraction](https://user-images.githubusercontent.com/1312469/209478179-85516d11-9701-4cc5-b608-f5691cab1d5c.png)
-
-### Metas and Tags
+Cardboard helps teams coordinate solving [puzzle hunts](https://en.wikipedia.org/wiki/Puzzle_hunt). It manages and organizes puzzles, keeps track of team members, integrates with other solving tools, and more.
 
-When adding a new puzzle, if you know it's a meta, you can mark it as such by checking the "Meta?" checkbox. This will automatically create a tag with the same name as the meta puzzle. You can then assign other puzzles as belonging to this meta by using the "Assign Metas" button for a puzzle or adding the meta puzzle tag using the "+" icon in the "Tags" column for a puzzle.
+For help **using Cardboard as a solver**, see [here](user-guide.md).
 
-The main puzzles page is grouped by metas, with puzzles belonging to a meta indented underneath the meta. Each puzzle may belong to multiple metas, in which case it will appear multiple times, once under each meta it belongs to.
+For help **setting Cardboard up for a hunt**, see [here](new-hunt-setup.md).
 
-Meta puzzles may also be assigned to other metas (which might be meta-metas).
+For information about **development**, see the [dev guide](dev-guide.md).
 
-In addition to meta tags for puzzles, users can also tag puzzles as high or low priority, logic or word puzzles, or create new tags.
+### Features
 
-### Google Sheets integration
+* Seamlessly supports (almost) any puzzle hunt structure
```

Comment:
(no action) "seamless" seems a bit confident lol, but I guess we have used it for several medium to large sized puzzle hunts :D

Reply:
Actually, on that note, would it make sense to list example puzzle hunts that we recommend using this tool for? e.g. MITMH, Puzzle Boat, etc.?